index.html

```diff
@@ -0,0 +1,334 @@
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </main>
+
+    <footer>
+      <div class="call-to-action">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in
+      </div>
+      <form name="contactform" action="https://formspree.io/f/xrgranbe" method="post">
+          <input type="text" name="name" placeholder="Full name" maxlength="30" required class="form-part-1" />
+          <input type="email" name="email" placeholder="Email address" required class="form-part-1" />
+          <input type="textarea" name="message" placeholder="Enter text here" class="message" maxlength="500" required />
+          <input type="submit" value="Get in touch" class="submit-btn" />
```

Comment:
- [ ] Firstly, you have introduced new codes that is outputing linter errors. Please, always make sure all linter errors are fixed before submitting your project for review. This is because the linters are configured to enforce best practices in our codes.
- [ ] Thank you for implementing `textarea` as requested by the previous reviewer. However, it's not well implemented. When done this way, you'll have just a single line input field that does not wrap texts. Consider doing something like this: ` <textarea rows="4" cols="50" name="comment" form="usrform">Enter text here..</textarea>`. This will provide a nicely formatted multi-line text area with a placeholder. Another reason is because one of the linters error you have has to do with the placeholder you have in your text area input the way you have implemented. Please let's fix it.